src/util/Affiliate.js

```diff
@@ -0,0 +1,15 @@
+
+function getTCGLink(card) {
```

Comment:
Might as well export this function as a named export, too.
```suggestion
export function getTCGLink(card) {
```